src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/suppliers.html.twig

```diff
@@ -0,0 +1,56 @@
+{# **
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * #}
+
+{% set suppliersForm = productForm.suppliers %}
+{% form_theme suppliersForm '@PrestaShop/Admin/Sell/Catalog/Product/Blocks/form_fields.html.twig' %}
+
+{% block form_product_suppliers %}
+  <div id="productSuppliers">
+    <div class="row">
+      <div class="col">
```

Comment:
You shouldn't handle the display field by field, just using `form_widget(suppliersForm)` should be enough, or the system we want to introduce to override/extend forms won't be appliable any more

Reply:
I agree, overwise, JS looks fine to me